go/vt/vtctl/grpcvtctldserver/topo.go

```diff
@@ -356,3 +363,55 @@ func removeShardCell(ctx context.Context, ts *topo.Server, cell string, keyspace
 
 	return err
 }
+
+// decodeContent decodes file data found at a given path in the topo
```

Comment:
there should be either a comment indicating this function was copied from `vtctl` (and should be kept in sync), or a way to code share